recipes/replxx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "replxx.hxx"
+
+int main(int argc, char **argv) {
+  using Replxx = replxx::Replxx;
+
+  Replxx rx;
```

Comment:
It would be nice having a method call here to validate the linkage. You can take something from the example https://github.com/AmokHuginnsson/replxx/blob/master/examples/cxx-api.cxx